### DISCUSSION TITLE
Chinese language

### DISCUSSION THREAD

**Discussion by Ptckh** [devemux86/cruiser#669] [Q&A] [answered]
Can I add new Languagde? How to do it?

**Comment by devemux86** [comment]
Hi @Ptckh,

Which language are you interested in?

- App language for user interface

- Voice language for navigation
(has much less texts)

**Reply by Ptckh** [reply]
App language for user interface.   Can I add Chinese？

**Reply by devemux86** [reply]
Simplified Chinese?

We could add a new language. There are some English text files to translate.
Do you want to translate them?

You can contact me in the email mentioned on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
Then I can prepare and send you the text files.

**Reply by Ptckh** [reply]
Yes. Simplified Chinese.

**Reply by Ptckh** [reply]
Yes. Simplified Chinese. Please send the file to gtrs.kong@gmail.com. I will work for this. Thank you.

**Reply by devemux86** [reply]
@Ptckh Thanks, give me some time to prepare the text files.

**Reply by devemux86** [reply]
@Ptckh

I sent you a zip with the text files for translation.
Thanks for your help! :+1:

**Comment by Ptckh** [comment]
Okay, Roger. I'll send it to you as soon as I'm done, thanks

**Comment by devemux86** [comment]
**Chinese** language and voice navigation will be available in the next Cruiser:

by @Ptckh

<img src="https://github.com/user-attachments/assets/7e424642-6630-4d63-be40-1c93c27e4829" width="50%"/><img src="https://github.com/user-attachments/assets/159a9ecc-d5e6-4adc-b82c-9813bd906387" width="50%"/>

**Comment by devemux86** [comment]
**Chinese** language and voice navigation is available in **[Cruiser 5.1.0-b5](https://github.com/devemux86/cruiser/discussions/671)**.

---

@Ptckh Thanks very much for the translation! :+1: 

Can you check if everything is ok?

**Comment by Ptckh** [comment]
Okay , isn't it the interface here
https://github.com/devemux86/cruiser/discussions/669, if it is, there is no
problem with this interface, it's great.

**Reply by devemux86** [reply]
@Ptckh

I tested by setting Chinese as Android system language or manually as app language.
Do you see something different?

Which Android version?
How do you set the language?
What have you selected as app language?
Which Chinese version?

**Comment by Ptckh** [comment]
Hi, SORRY, I HAVEN'T USED THE ANDROID VERSION OF CRUISER, I USE IT ON
WIN11. My Android is 11 12 and my PC is WIN11 I usually use the Chinese
Simplified Chinese version. I haven't had time to see it just now, but now
I'm looking at WIN to see if there's a problem.

**Comment by Ptckh** [comment]
Hi,  Under WIN11, the context menu seems to be missing Add via  Points and
shaping Points and Clear ,There doesn't seem to be a bug elsewhere for the
time being. Very Well.

**Reply by devemux86** [reply]
@Ptckh Which menu, can you post a screenshot?

I see everything translated in this menu:

![Cruiser](https://github.com/user-attachments/assets/00d0f57b-934a-49e8-aaa8-8f7e2192f595)

**Comment by Ptckh** [comment]
Right-click menu, I just have a screenshot of the attachment

**Reply by devemux86** [reply]
@Ptckh Better use the GitHub website in a browser to attach screenshots.

**Comment by Ptckh** [comment]
OK, now

**Reply by Ptckh** [reply]
![1731836463131](https://github.com/user-attachments/assets/497f70a9-73a2-48a1-906c-6200b0d7ffbb)

**Reply by devemux86** [reply]
@Ptckh On what have you right clicked? On empty map area?

**Reply by devemux86** [reply]
@Ptckh This menu should appear when there is no route.

Via / shaping points can be added after a route is calculated.

**Comment by Ptckh** [comment]
I'm right-clicking on the map and I want to add a via point, or a shape
point, but I can't find a menu item

**Reply by devemux86** [reply]
@Ptckh First must calculate a route with start + end points.
Then the via / shaping menu is available to add via / shaping points.

**Comment by Ptckh** [comment]
ok, I see. thanks. I tested the Android version in WSA and I haven't found
any issues so far. It's great.

**Comment by devemux86** [comment, marked as answer]
- Chinese language and voice navigation is available in https://github.com/devemux86/cruiser/discussions/684

@Ptckh Thanks again for the translations! :+1:

**Comment by Ptckh** [comment]
You are Welcome.


---- Replied Message ----
| From | ***@***.***> |
| Date | 12/13/2024 19:09 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [devemux86/cruiser] Chinese language (Discussion #669) |
Chinese language and voice navigation is available in #684

@Ptckh Thanks again for the translations! 👍
